drivers/serial/uart_stm32.h

```diff
@@ -19,6 +19,8 @@ struct uart_stm32_config {
 	struct stm32_pclken pclken;
 	/* Baud rate */
 	u32_t baud_rate;
+	/* HW flow control */
+	u32_t rts_cts:1;
```

Comment:
"config" struct is device startup configuration.
You should not use it to store information that will be modified at run time.

Reply:
yes, it's my mistake and actually 'config' is const and can't changed.

Reply:
I want move baud_rate from uart_stm32_config to uart_stm32_data. Because uart_stm32_data can be changed at runtime. what's your opinion?

Reply:
It should be ok to do that.
But what about setting/reading directly in registers?

Reply:
change baud rate need disable usart first and then calc clock rate and then set registers. thats was almost did in stm32_init.